inc/class-s3-uploads-stream-wrapper.php

```diff
@@ -17,7 +17,34 @@ public static function register( Aws\S3\S3Client $client)
 		static::$client = $client;
 	}
 
+	// Override
+	public function stream_flush() {
+
+		// Add some HTTP headers which can be defined in wp-config.php
+
+		// Expires:
+		if ( defined( 'S3_UPLOADS_EXPIRES' ) ) {
+			$this->params[ 'Expires' ] = S3_UPLOADS_EXPIRES;
```

Comment:
This class extends the parent which already defines `$params` 

If I `var_dump` at the beginning of our `stream_flush()`, the `$params` var is setup and contains 

```
array(3) {
  ["Bucket"]=>
  string(7) "s3-bucket-name"
  ["Key"]=>
  string(26) "uploads/2015/06/stock5.jpg"
  ["ACL"]=>
  string(11) "public-read"
}
```

So I don't think we'd need to add that in, right?